### PR TITLE
Add minimum and maximum ratio for the Balloon

### PR DESCRIPTION
**Pull request by JoePaul**
Hope everything is good :) Got some more additions I would like to add. These additions makes it possible to add a min and max ratio, which works just like the min and max for pixels but with ratio instead. I also fixed so it should work with the new text measuring  implementation.

### CODE REVIEW

File: balloon/src/main/java/com/skydoves/balloon/Balloon.kt [hunk 1]
```diff
@@ -1219,6 +1223,10 @@ class Balloon(
     return when {
       builder.widthRatio != NO_Float_VALUE ->
         (displayWidth * builder.widthRatio).toInt() - spaces
+      builder.minWidthRatio != NO_Float_VALUE || builder.maxWidthRatio != NO_Float_VALUE -> {
+        val max = if (builder.maxWidthRatio != NO_Float_VALUE) builder.maxWidthRatio else 1f
+        binding.root.measuredWidth.coerceIn((displayWidth * builder.minWidthRatio).toInt(), (displayWidth * max).toInt())
```
Comment (skydoves) [inline balloon/src/main/java/com/skydoves/balloon/Balloon.kt:1228]: Missed `- spaces`.

**Review by skydoves — CHANGES_REQUESTED** [review]
Thanks for your contributions again!
I left a comment related to a small thing.

**Comment by skydoves** [comment]
Oh, sorry for making a confusion.
I meant this way:

```kotlin
binding.root.measuredWidth.coerceIn((displayWidth * builder.minWidthRatio).toInt(), (displayWidth * max).toInt()) - spaces
```

**Comment by JoePaul** [comment]
It's my bad, I hopefully fixed it now :)

**Review by skydoves — APPROVED** [review]
LGTM 👍 
Thanks for your contribution! 😄

**Pull request merged**